source/documentation/using_the_api/api_reference.md

```diff
@@ -374,21 +358,17 @@ You can download the new entries in 2 ways:
 * download another copy of the full register
 * download individual updated entries for the records you are using
 
-### Check for updates
-
-Path: GET /register
+### GET /register
 
 You can find the latest entry number by looking at the register information and comparing the most recent entry number with your own copy.
 
 Example request: https://country.register.gov.uk/register
 
-### Download a full new copy of the register
+### <a name="downloadreg"></a>GET /download-register
```

Comment:
Why some headers have an anchor and others don't?

Reply:
I left those in because they were in the original; I assumed those were linked to elsewhere (which seems to be the case) but there's no reason not to add anchors to all headers, anyway. The only reason I have refrained from opening a PR to add anchors to do that yet is because we're still changing them.